### PR TITLE
Add `run-operation` command to `run_results.json`

### PR DESCRIPTION
**Pull request by aranke**
## What are you changing in this pull request and why?

Adding `run-operation` to `run_results.json`; this was an oversight but will be fixed in dbt 1.6.

`dbt-core` merged PR: https://github.com/dbt-labs/dbt-core/pull/7655

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/docs/reference/artifacts/run-results-json.md [hunk 1]
```diff
@@ -5,7 +5,28 @@ sidebar_label: "Run results"
 
 **Current schema**: [`v4`](https://schemas.getdbt.com/dbt/run-results/v4/index.html)
 
-**Produced by:** [`build`](/reference/commands/build) [`compile`](/reference/commands/compile) [`docs generate`](/reference/commands/cmd-docs) [`seed`](/reference/commands/seed) [`snapshot`](/reference/commands/snapshot) [`test`](/reference/commands/test) [`run`](/reference/commands/run)
+<VersionBlock lastVersion="1.6">
```
Comment (runleonarun) [inline website/docs/reference/artifacts/run-results-json.md]: Hey @aranke I wonder if this should be 1.5? Otherwise both blocks might show up?

Comment (runleonarun) [reply website/docs/reference/artifacts/run-results-json.md]: Also, I wonder if you could version only the new command for 1.6 like this (which might make the code easier to read):
```
**Produced by:**
  [`build`](/reference/commands/build)
  [`compile`](/reference/commands/compile)
  [`docs generate`](/reference/commands/cmd-docs)
  [`run`](/reference/commands/run)
 <VersionBlock firstVersion="1.6"> [`run-operation`](/reference/commands/run-operation)</VersionBlock>
  [`seed`](/reference/commands/seed)
  [`snapshot`](/reference/commands/snapshot)
  [`test`](/reference/commands/test)
```

Comment (aranke) [reply website/docs/reference/artifacts/run-results-json.md]: Done, thanks!

Comment (mirnawong1) [reply website/docs/reference/artifacts/run-results-json.md]: going to fix this as it's looking like this and snapshots, test, seeds don't appear correctly in v1.5 and lower.
 
Screenshot 2023-06-07 at 12 02 33

**Review by mirnawong1 — APPROVED** [review]
fixed versioning format and it's looking correct in the deploy preview, approving and merging

**Comment by mirnawong1** [comment]
merging since this look correct

Screenshot 2023-06-07 at 13 12 43

**Comment by mirnawong1** [comment]
hey @aranke and @runleonarun, fyi i've merged this pr for ya'll

**Pull request merged**